docs/users-guide.md

```diff
@@ -1,3 +1,78 @@
 # OpenShift Service Mesh Console User's Guide
 
-TODO
+The OpenShift Service Mesh Console (aka OSSMC) is an extension to the OpenShift Console which provides visibility into your Service Mesh. The functionality you see described here is very similar to that of the standalone Kiali Console (see https://kiali.io). In fact, you can still access the standalone Kiali Console if you wish. This User's Guide, however, will discuss the functionality that you see from within the OpenShift Console itself.
```

Comment:
It'd be helpful adding a blurb here saying that the plugin both provides its own "tab" (or whatever the left side nav section is called) and enhances existing ones like the "Workloads" pages in order to set expectations on what the plugin does.

Reply:
done.